crates/topos-p2p/src/tests/dht.rs

```diff
@@ -40,7 +41,14 @@ async fn put_value_in_dht() {
     _ = kad
         .inner
         .put_record(
-            Record::new(input_key.clone(), runtime.addresses.to_vec()),
+            Record::new(
+                input_key.clone(),
+                runtime
+                    .advertised_addresses
+                    .first()
+                    .map(Multiaddr::to_vec)
+                    .unwrap(),
```

Comment:
No unwrap please

Reply:
It is in tests and the `advertised_addresses` are defined just above so it can't fail.